fplll/nr/matrix.h

```diff
@@ -79,7 +79,7 @@ template <class T> class MatrixRow
   MatrixRow(const NumVect<T> &row) : row(const_cast<NumVect<T> &>(row)) {}
   NumVect<T> &row;
 };
-
+/** Computes the dot product between two rows of a Matrix */
```

Comment:
I think Matrix should be lowercase here as it refers to the mathematical object not the class name, right?